examples/api/test/widgets/routes/flexible_route_transitions.0_test.dart

```diff
@@ -0,0 +1,18 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_api_samples/widgets/routes/flexible_route_transitions.0.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('Flexible Transitions App is able to build', (WidgetTester tester) async {
```

Comment:
You should verify that the VerticalTransitionPageRoute and the route below it both use the vertical transition. Maybe look for VerticalPageTransition in the widget tree? Or let the animation run halfway and expect that the pages have moved vertically?

Reply:
Thanks, this is done now :+1: